crates/swc_ecma_parser/tests/jsx/errors/13/input.js.stderr

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
This file change is expected, the error is the same as typescript, https://www.typescriptlang.org/play?#code/DwGQpgZgLgSglgcwBZQAQBtJQLzAIaoD0AfKgE6Iq4BGxAtgPYB2A1mAJ4DORpCDF6dHk7BCtHkA

Reply:
But it's js, not ts.

https://babeljs.io/repl#?browsers=node%200.8&build=&builtIns=false&corejs=false&spec=false&loose=false&code_lz=DwGQpgZgLgSglgcwBZQAQBtJQLzAIaoD0AfKgE6Iq4BGxAtgPYB2A1mAJ4DORpCDF6dHk7BCtHkA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Creact&prettier=false&targets=&version=7.19.0&externalPlugins=&assumptions=%7B%7D

Reply:
Babel will not emit any error even in tsx

Reply:
So do we have the ability to emit a recoverable error?

Reply:
Yes, `emit_err` exists for it